File: support_tools/swigwin-3.0.12/Examples/test-suite/ruby/template_extend2_runme.rb
#!/usr/bin/env ruby
#
# Put description here
#
# 
# 
# 
#

require 'swig_assert'

require 'template_extend2'

a = Template_extend2::LBaz.new
b = Template_extend2::DBaz.new

raise RuntimeError unless a.foo() == "lBaz::foo"
raise RuntimeError unless b.foo() == "dBaz::foo"

File: support_tools/swigwin-3.0.12/Examples/test-suite/go/template_extend2_runme.go
package main

import "./template_extend2"

func main() {
	a := template_extend2.NewLBaz()
	b := template_extend2.NewDBaz()

	if a.Foo() != "lBaz::foo" {
		panic(0)
	}

	if b.Foo() != "dBaz::foo" {
		panic(0)
	}
}
